ocaml/xapi/sync_networking.ml

```diff
@@ -148,6 +148,23 @@ let copy_bonds_from_master ~__context () =
 (* This is now executed fully on the master, once asked by the slave when the slave's Xapi starts up *)
 let copy_vlans_from_master ~__context () =
   let host = !Xapi_globs.localhost_ref in
+  let master = Helpers.get_master ~__context in
```

Comment:
Currently Pool join doesn't put the management on VLAN PIF. In case of VLAN PIF having static IP configuration, Pool join can't retain it until it is introduced before hand.